source/common/formatter/stream_info_formatter.cc

```diff
@@ -1428,6 +1428,26 @@ const StreamInfoFormatterProviderLookupTable& getKnownStreamInfoFormatterProvide
             [](const std::string& key, absl::optional<size_t> max_length) {
               return std::make_unique<EnvironmentFormatter>(key, max_length);
             }}},
+          {"UPSTREAM_CONNECTION_POOL_CALLBACK_DURATION",
+           {CommandSyntaxChecker::COMMAND_ONLY,
+            [](const std::string&, const absl::optional<size_t>&) {
+              return std::make_unique<StreamInfoDurationFormatterProvider>(
+                  [](const StreamInfo::StreamInfo& stream_info) {
+                    return stream_info.upstreamInfo().has_value() &&
+                                   stream_info.upstreamInfo()
+                                       .value()
+                                       .get()
+                                       .upstreamTiming()
+                                       .connectionPoolCallbackLatency()
+                                       .has_value()
+                               ? stream_info.upstreamInfo()
+                                     .value()
+                                     .get()
+                                     .upstreamTiming()
+                                     .connectionPoolCallbackLatency()
+                               : absl::nullopt;
```

Comment:
One line style is great. But may using simple if check is better at here?

```
if(auto upstream_info = ...; upstream_info.has_value) {
  if(...) {
  }
}
return absl::nullopt;
```

Reply:
done, PTAL, thanks.